cypress/e2e/auth.cy.ts

```diff
@@ -16,8 +40,24 @@ describe("Authentication tests", () => {
     it("Sign out", () => {
         cy.login();
         cy.visit("/clients");
+        cy.url().should("include", "/clients");
         cy.get("button[aria-label='Sign Out Button']").click();
 
         cy.url().should("include", "/login");
     });
+
+    it("CSR Redirected to login page", () => {
```

Comment:
is this not the same as `Redirected to login page` tests? especially if we don't start off in a logged in state

Reply:
No, because we wait for hydration and click `Link`s.

This ensures it uses client side routing and covers our bases better. With how it's currently implemented there shouldn't be a difference, but having the test ensures it stays that way.